docs/emcc.txt

```diff
@@ -141,9 +141,13 @@ Options that are modified or new in *emcc* are listed below:
 "-gseparate-dwarf[=FILENAME]"
    Preserve debug information, but in a separate file on the side.
    This is the same as "-g", but the main file will contain no debug
-   info, while debug info will be present in a file on the side
-   ("FILENAME" if provided, otherwise the same as the wasm file but
-   with suffix ".debug.wasm").
+   info, while debug info will be present in a file on the side,
```

Comment:
this sentence is quite long and could be split up

Reply:
Done.